tests/snuba/api/endpoints/test_project_group_index.py

```diff
@@ -352,6 +352,36 @@ def test_filter_not_unresolved(self):
         assert response.status_code == 200
         assert [int(r["id"]) for r in response.data] == [event.group.id]
 
+    def test_group_by_hash(self):
```

Comment:
Nit:

```suggestion
    def test_single_group_by_hash(self):
```

I'd do `test_multiple_groups_by_hashes` below, too - just so it doesn't sound like "group" is a verb and like you're testing it doing grouping.